tests/lua/lua-base64/output.lua

```diff
@@ -0,0 +1,43 @@
+-- Test that "suricata.base64" can be used from a Lua output
+-- script. More thourough testing of base64 in rule.lua.
+
+local base64 = require("suricata.base64")
+
+local expected_base64 = "d3d3LnN1cmljYXRhLWlkcy5vcmc="
+
+filename = "results.log"
+
+function init (args)
+   local needs = {}
+   needs["protocol"] = "dns"
+   return needs
+end
+
+function setup (args)
+   SCLogNotice("lua: setup()")
+   file = assert(io.open(SCLogPath() .. "/" .. filename, "w"))
```

Comment:
I did not know this lua output functionality

Is it not limited by the sandbox ? (using io to open some arbitrary file)

Reply:
The output modules are not sandbox restricted.

Reply:
TIL thanks